release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -199,6 +199,13 @@ For more information, see xref:../nodes/scheduling/nodes-scheduler-pod-topology-
 
 With this release, the `MaxUnavailableStatefulSet` featureset configuration parameter is enabled by default. This allows users to define the maximum number of `StatefulSet` pods that can be unavailable during updates, and reduces application downtime when upgrading.
 
+[id="ocp-4-13-pdb-eviction-policy"]
+==== Pod disruption budget (PDB) unhealthy pod eviction policy
+
+With this release, specifying an unhealthy pod eviction policy for pod disruption budgets (PDBs) is generally available in {product-title}. This helps evict malfunctioning applications during a node drain. There is no new functionality in the GA version.
```

Comment:
This feature is not GA yet. It is still beta https://github.com/openshift/kubernetes/blob/f815d27efda36ebb7bc87dd642cf4811b2352f5e/pkg/features/kube_features.go#L625

Reply:
@atiratree PTAL at the changes, thanks!

Reply:
@cbippley sorry for the confusiong, but we have to use OCP terminology instead: https://issues.redhat.com/browse/WRKLDS-707?focusedId=22877875&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22877875

Reply:
@atiratree thanks !! And just wanted to confirm that the upstream tests what we have as of today is sufficient right to make this product GA ?